src/cascadia/TerminalSettingsEditor/ColorSchemesPageViewModel.idl

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "ColorSchemeViewModel.idl";
+
+namespace Microsoft.Terminal.Settings.Editor
+{
+    runtimeclass ColorSchemesPageViewModel : Windows.UI.Xaml.Data.INotifyPropertyChanged

```

Comment:
Since the page `Editor::ColorSchemes` only ever talks to this VM, I felt like having the word `Page` fit here (like its the VM for the entire page)... If you feel strongly about it I can change it but to me `List` doesn't quite encapsulate what this VM is responsible for since tracking the currently selected item/renaming items feels more than what a list does

Reply:
eh, fair enough haha